bin/upgrade.py

```diff
@@ -160,9 +160,9 @@ def upgrade_statement(problem_path: Path, bar: ProgressBar) -> None:
     if (problem_path / "problem_statement").is_dir():
         if (problem_path / "statement").exists():
             bar.error("can't rename 'problem_statement/', 'statement/' already exists", resume=True)
-            return
-        bar.log("renaming 'problem_statement/' to 'statement/'")
-        (problem_path / "problem_statement").rename(problem_path / "statement")
+        else:
+            bar.log("renaming 'problem_statement/' to 'statement/'")
+            (problem_path / "problem_statement").rename(problem_path / "statement")
 
     origin = problem_path / "statement"
     move = [
```

Comment:
Talking about images, we _could_ detect which images are only used in `solution.en.tex` and move those to `solution/` as well, but that's much more effort than needed, and things will just work when all the images are in `statement/` :joy:

Reply:
yeah... best effort service is enough here :p